plugin/src/main/java/io/fabric8/maven/plugin/mojo/build/BuildMojo.java

```diff
@@ -17,6 +17,7 @@
 package io.fabric8.maven.plugin.mojo.build;
 
 
+import java.awt.*;
```

Comment:
If possible can we import specific packages?

Reply:
Oops, good point. That shouldn't be there at all ....

Reply:
@metacosm btw, the bogus import has been already removed within this PR at --> https://github.com/fabric8io/fabric8-maven-plugin/pull/1112/commits/a5a72194e8789812a048145e1fffc46839950c56